### ISSUE TITLE
build server logs error when a translator resubmits a translation

### ISSUE THREAD

**Issue by jbphet** [phetsims/perennial#4]
Currently, if a user submits a translation, then makes some mods and resubmits, the build server will log an error in the log file.  This error indicates that there is already an entry for this user-sim-locale combination.  Since this is a normal condition, it would be good to keep the logs clean and either check the DB first or catch the error.

The error looks like this:

```
12437 2015-11-20T14:08:51.540Z - info: running SQL command: INSERT INTO user_localized_simulation_mapping SELECT localized_simulation.id, phet_user.id from localized_simulation, simulation, project, phet_      user WHERE simulation.name = 'ph-scale' AND locale = 'da' AND phet_user.id = 130982 AND simulation = simulation.id AND simulation.project = project.id AND project.type = 2
12438 2015-11-20T14:08:51.557Z - error: adding to user_localized_simulation_mapping, translator probably already exists for this sim and locale
12439 2015-11-20T14:08:51.557Z - error:  error: duplicate key value violates unique constraint "user_localized_simulation_mapping_pkey"
12440     at Connection.parseE (/data/share/phet/phet-repos/perennial/node_modules/pg/lib/connection.js:539:11)
12441     at Connection.parseMessage (/data/share/phet/phet-repos/perennial/node_modules/pg/lib/connection.js:366:17)
12442     at Socket.<anonymous> (/data/share/phet/phet-repos/perennial/node_modules/pg/lib/connection.js:105:22)
12443     at Socket.emit (events.js:95:17)
12444     at Socket.<anonymous> (_stream_readable.js:764:14)
12445     at Socket.emit (events.js:92:17)
12446     at emitReadable_ (_stream_readable.js:426:10)
12447     at emitReadable (_stream_readable.js:422:5)
12448     at readableAddChunk (_stream_readable.js:165:9)
12449     at Socket.Readable.push (_stream_readable.js:127:10)
```

**Comment by jonathanolson** [comment]
@jbphet, should this be done for chipper 2?

**Comment by jbphet** [comment]
Yes, this would be a good thing to have on the list.

**Comment by jbphet** [comment]
This is an old issue and the build server has evolved quite a bit since the time this issue was created.  I just looked through the build server log on phet-server, which currently has about 5 1/2 days worth of data in it, and I see no signs of this message.  Also, I searched through the build sever code for the string "translator probably" from the error message shown above and couldn't find it in the code.  Thus, I think it's safe to say that we no longer need to address this issue.  Closing.
